tensix/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.10)
```

Ubuntu 20.04 is EOL'd this month and 22.04 comes with CMake v3.22 by default.  So that's a pretty reasonable minimum to use for new CMake files these days.  Pretty much every system should have at least that version.

And for forward compatibility, it's a good idea to add `...3.x` where x is the version you tried things out on.  It affects CMake policies and will quiet down deprecation warnings since CMake knows you're not relying on any deprecated behaviour/bugs.

```suggestion
cmake_minimum_required(VERSION 3.22...3.30)
```

---

Will do. I had originally put 20.04's version in there because I saw some people were still referencing it with metalium.

---

Yes, we're a little late to the party getting off of 20.04.  It's been active work.

---

On a semi-relevant side note, I am on fedora and as of fedora41 they are providing risc-v ISOs.